build/ublue-os-just/00-default.just

```diff
@@ -31,11 +31,10 @@ regenerate-grub:
 
 # Enroll Nvidia driver & KMOD signing key for secure boot - Enter password "universalblue" if prompted
 enroll-secure-boot-key:
-    echo 'Enter password "universalblue" if prompted after your user password.'
     sudo mokutil --timeout -1
+    echo 'Enter password "universalblue" if prompted next'
     sudo mokutil --import /etc/pki/akmods/certs/akmods-ublue.der
-    echo 'When you reboot your computer, follow the instructions to start MOK util'
-    echo 'by pressing a key, then enroll the secure boot key and enter "universalblue" as the password'
+    echo 'At next reboot, the mokutil UEFI menu UI will be displayed (*QWERTY* keyboard input and navigation).\nThen, select "Enroll MOK", and input "universalblue" as the password'
```

Comment:
This is a good improvement. However, the ` (*QWERTY* keyboard input and navigation)` section may not be clear and may be too much detail. Please remove that section. Thanks!

Reply:
It was the reason I originally opened that PR actually

In my zone, AZERTY is the standard, and it took me hours to understand why the password were not matching - ARERTY("universalblue") == QWERTY("universqlblue")

It seems to have bitten other people too;

See https://universal-blue.discourse.group/t/mokutil-list-enrolled-explanation/4266/18?u=malix

How do you suggest this information should be included better ?

Reply:
Hmm... apologies, as I'd missed the detail around that topic.

The MOK import is problematic for a number of reasons. This is another one added to the list. :-(

If the script always set the MOK password to "universalblue" (QWERTY)  does that help? 

I assume it would not, as likely the problem is the UEFI/MOK firmware not respecting keyboard layouts?

Reply:
> the problem is the UEFI/MOK firmware not respecting keyboard layouts?

The problem indeed comes from the mokutil UEFI menu UI not respecting keyboard layouts (always QWERTY)

So for example, my AZERY "a" is translated as a QWERTY "q" there (so "universqlblue")

And as the password input UI doesn't let you see character, it was impossible to know before [someone on the forum told me it could come from there](https://universal-blue.discourse.group/t/mokutil-list-enrolled-explanation/4266/18?u=malix), after having wasted 15 minutes trying with different methods :sweat_smile:

Reply:
i wonder if setting the password to `ublue` will be good enough to avoid issues with AZERTY and QWERTZ layouts

Reply:
@HikariKnight 

That would work but `ublue` is too short. I think the minimum is 8 chars?